lib/components/Input/NumberInput.tsx

```diff
@@ -226,12 +226,6 @@ export class NumberInput extends React.Component<NumberInputProps, NumberInputSt
         }
     }
 
-    UNSAFE_componentWillReceiveProps(newProps: NumberInputProps) {
```

Comment:
don't we need the getDerivedStateFromProps here?

Reply:
No, the numberInput already manages it's own state... This cwrp is allowing the user to update the initialValue without user interaction (i.e. the user changes another field and the number field is updated).

With this change that scenario won't be supported anymore, but as far as I understand we don't do that type of behavior anywhere, so this change should be safe. Another thing I could do is move this logic to componentDidUpdate and update the state there, but this would be updated a rerender later than how it use to. 

Which solution do you think is better?

Reply:
Moved it to componentDidUpdate